act-rules-format.bs

```diff
@@ -695,26 +695,29 @@ Examples in this section include:
   "@context": "context.json",
   "@type": "Assertion",
   "assertedBy": {
-    "@type": "foaf:Organization",
-    "@id": "_:myOrg"
-  }
+    "@type": "Organization",
+    "@id": "_:myOrg",
+    "title": "My Organization",
+    "description" : "Accessibility testing service",
+    "homepage" : "http://example.org/myOrg/"
+  },
   "subject": {
     "@type": ["WebSite", "TestSubject"],
     "@id": "https://example.org/"
-  }
+  },
   "test": {
     "@type": "earl:TestRequirement",
     "@id": "http://www.w3.org/WAI/WCAG2A-Conformance"
   },
   "result": {
-    "outcome": "Failed",
+    "outcome": "earl:failed",
     "source": [{
       "test": {
         "@type": "earl:TestRequirement",
         "@id": "WCAG21:non-text-content"
       },
       "result": {
-        "outcome": "Failed",
+        "outcome": "earl:failed",
```

Comment:
These need to be IDs, without a namespace they don't map. It's a quirky thing in JSON-LD.